openhands/cli/mypy.ini

```diff
@@ -0,0 +1,2 @@
+[mypy]
+warn_unreachable = False
```

Comment:
Does this disable mypy warnings when statements are unreachable? If so we should remove it